Documentation/contributing.rst

```diff
@@ -1482,27 +1482,71 @@ GitHub issues using the process below:
 Triage process
 ^^^^^^^^^^^^^^
 
-#. Investigate the failure
-#. Search `GitHub issues <https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=is%3Aissue+>`_
-   to see if bug is already filed. Make sure to also include closed issues in
-   your search as a CI issue can be considered solved and then re-appears.
-#. If no existing GitHub issues was found, file a new GitHub issue:
+#. Discover Jenkins failures via the jenkins-failures.sh script. It defaults to
+   the previous 24 hours but this can be modified by setting the SINCE
+   environment variable (it is a unix timestamp). The script checks the various
+   test pipelines that need triage.
+::
+
+    $ contrib/scripts/jenkins-failures.sh
+
+.. note::
+
+   You can quickly assign SINCE with statements like ``SINCE=`date -d -3days```
+
+#. Investigate the failure you are interested in and determine if it is a
+   CI-Bug, Flake, or a Regression.
+
+   #. Search `GitHub issues <https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=is%3Aissue+>`_
+      to see if bug is already filed. Make sure to also include closed issues in
+      your search as a CI issue can be considered solved and then re-appears.
+      Good search terms are:
+
+      - The test name, e.g.
+      ::
+
+          k8s-1.7.K8sValidatedKafkaPolicyTest Kafka Policy Tests KafkaPolicies (from (k8s-1.7.xml))
+
+      - The line on which the test failed, e.g.
+      ::
+
+          github.com/cilium/cilium/test/k8sT/KafkaPolicies.go:202
+
+      - The error message, e.g.
+      ::
+
+          Failed to produce from empire-hq on topic deathstar-plan
+
+#. If a corresponding GitHub issue exists, update it with:
+
+   #. A link to the failing Jenkins build (note that these will expire).
```

Comment:
Specify what 'expire' means - i.e., that we do not save builds.

Reply:
good point. I'll explicitly say deleted